gcp/website/frontend_handlers.py

```diff
@@ -55,6 +55,10 @@
 _BLOG_CONTENTS_DIR = 'blog'
 _DEPS_BASE_URL = 'https://deps.dev'
 _FIRST_CVSS_CALCULATOR_BASE_URL = 'https://www.first.org/cvss/calculator'
+_GO_VANITY_METADATA = \
+  ('<meta name="go-import" '
+   'content="osv.dev git https://github.com/google/osv.dev">')
```

Comment:
Can we add a check so this is only on prod?
I don't know what would happen if `test.osv.dev` would return this.

Reply:
I think it's fine for the test to return this? it will just fail to install, since package name would not match, and this way we can test the responses while in the test instance.